websocket-ballerina/Module.md

```diff
@@ -0,0 +1,44 @@
+## Module Overview
```

Comment:
I think it is good to have two high level sections as `Client` and `Listener`. Also few topics which we can add into this is 

Concurrency
Compression
Origin Considerations

Reply:
Added details as requested. Please have a look.